python/fledge/plugins/south/mqtt-readings/mqtt-readings.py

```diff
@@ -130,11 +130,18 @@
         'minimum': '0',
         'maximum': '2'
     },
+    'json_object_key': {
+        'description': 'If only a value is sent this needs to be converted to an json object, the value given here will he the key for the value in that json object',
+        'type': 'string',
+        'default': 'value',
+        'order': '8',
+        'displayName': 'Json Object Key',
```

Comment:
I would suggest to name it `Datapoint Name` and description to be set accordingly e.g. `Datapoint name to be used in the reading object only for a primitive value, published to the topic`


Config item key also can be named accordingly to make that self explanatory, inside the code e.g. `reading_datapoint_name_for_primitive_value`
 


For nomenclature see https://fledge-iot.readthedocs.io/en/develop/glossary.html?highlight=datapoint#term-Datapoint

Reply:
that is a more descriptive name, thanks